canvas_account_admin_tools/tests/jasmine/shared/SpecHelpers.js

```diff
@@ -0,0 +1,34 @@
+var spec = {};
+
+spec.diSanityCheck = function(providerList) {
```

Comment:
Actually, `di` stands for `dependency injection` -- I can certainly add some docstring in there, thanks!